lib/ColorPicker-view.coffee

```diff
@@ -262,7 +262,7 @@
             EditorElement = Editor.getElement()
 
             return unless EditorView
-            EditorRoot = EditorView.shadowRoot or EditorView
+            EditorRoot = EditorView
```

Comment:
Could you, instead of creating a duplicate reference to the same variable, change the subsequent references from `EditorRoot` to `EditorView`?